### PR TITLE
Update station name per the project's recent renaming

### PR DESCRIPTION
**Pull request by machawk1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,5 +44,5 @@ This is the internal repo to for Project Management of the IPFS GUI project.
   - [Design Tools Research](https://docs.google.com/document/d/1qJyfwgcMg8l3Tk3aYxF38iyYRhkEf3nlLNqOw4ZiW_8/edit)
 - Code
   - [webui](https://github.com/ipfs/webui)
-  - [station](https://github.com/ipfs-shipyard/station)
+  - [desktop](https://github.com/ipfs-shipyard/desktop)
```
Comment (lidel) [inline README.md]: Oopw Error 404.  Should be `ipfs-desktop` :)

Comment (machawk1) [reply README.md]: Good catch. Fixed.

File: README.md [hunk 2]
```diff
@@ -44,5 +44,5 @@ This is the internal repo to for Project Management of the IPFS GUI project.
   - [Design Tools Research](https://docs.google.com/document/d/1qJyfwgcMg8l3Tk3aYxF38iyYRhkEf3nlLNqOw4ZiW_8/edit)
 - Code
   - [webui](https://github.com/ipfs/webui)
```
Comment (diasdavid) [inline README.md]: While you are at it, wanna fix also this link to https://github.com/ipfs-shipyard/ipfs-webui ?

Comment (diasdavid) [reply README.md]: I went ahead and did it :)

**Pull request merged**